From 17aeeb39246759a4e9c7fcd20690921de6eb5880 Mon Sep 17 00:00:00 2001
From: Emmanuele Bassi <ebassi@gnome.org>
Date: Mon, 8 Feb 2016 14:09:35 +0000
Subject: [PATCH] Add autogen.sh

The autogen.sh name is widely used in X.org, GNOME, and many others.

http://people.gnome.org/~walters/docs/build-api.txt
---
 autogen.sh | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100755 autogen.sh

diff --git a/autogen.sh b/autogen.sh
new file mode 100755
index 0000000..ad94809
--- /dev/null
+++ b/autogen.sh
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec make bootstrap
-- 
2.5.0

